src/main/java/picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.java

```diff
@@ -216,7 +216,7 @@ protected SamHeaderAndIterator openInputs(boolean eagerlyDecode) {
         final List<SamReader> readers = new ArrayList<>(INPUT.size());
 
         for (final String input : INPUT) {
-            SamReaderFactory readerFactory = SamReaderFactory.makeDefault();
+            SamReaderFactory readerFactory = SamReaderFactory.makeDefault().referenceSequence(REFERENCE_SEQUENCE);
```

Comment:
I assume this has been tested and is confirmed to be working? I don't know that there are any integration tests for MarkDuplicates where a cram could be added.